.github/workflows/newsletter-issue.yml

```
@@ -0,0 +1,33 @@
+name: Newsletter Issue
+
+on:
+  schedule:
+  - cron: "30 0 1 * *"    # At 00:30 on day-of-month 1.
```

```suggestion
    - cron: "30 0 1 * *"    # At 00:30 on day-of-month 1.
```
Just for consistency :smile: